README.md

```diff
@@ -1,20 +1,29 @@
 # digital_ingest_pipeline
-Pipeline for ingesting digitized and born digital records into preservation systems.
+Pipeline for ingesting digitized and born digital records into preservation systems, and delivering accessions, archival objects, and digital objects to ArchivesSpace.
 
 ## Usage
 
-This repository is intended to be deployed in AWS infrastructure.
+This repository contains AWS CloudFormation templates that create the infrastructure for a
+pipeline designed to integrate with [Aurora](https://github.com/RockefellerArchiveCenter/aurora), 
+[ArchivesSpace](https://archivesspace.org/), and [Archivematica](https://www.archivematica.org/en/).
 
-To provision, authenticate as a user with the necessary permissions, 
-update variables and run the following command:
+## Architecture Diagram
 
-aws cloudformation deploy \
-	--template-file digital_ingest_pipeline_template.yaml \
-	--stack-name digital-ingest-pipeline \
-	--capabilities CAPABILITY_AUTO_EXPAND CAPABILITY_NAMED_IAM \
-	--parameter-overrides \
-		ArchivesSpacePassword={password for archivesspace user} \
+![Overview diagram of digital ingest pipeline, showing how individual services are connected](digital_ingest_pipeline.png)
+
+## Related Repositories
+
+- [digital_ingest_assembly](https://github.com/RockefellerArchiveCenter/digital_ingest_assembly/) - creates Archivematica-compliant Submission Information Packages (SIPs).
+- [digital_ingest_discovery](https://github.com/RockefellerArchiveCenter/digital_ingest_discovery/) - discovers 
+and prepares the SIPs for processing.
```

Comment:
Are these SIPs at this point? Or do they become SIPs after the assembly?

Reply:
You're right, they're technically only SIPs once they get started in Archivematica.